### PR TITLE
Removes pain and brute damage from ERP menu

### PR DESCRIPTION
**Pull request by K4rlox**
## About The Pull Request

removes taking bleeding and brute damage from ERP menu
## Why It's Good For The Game

its retarded
## Changelog
:cl:
fix: you no longer take brute damage from sex
/:cl:

### CODE REVIEW

**Comment by EaglePhntm** [comment]
No, removing it all is lame, and yet you still havent found the issue you spoke of before.

**Comment by K4rlox** [comment]
> No, removing it all is lame, and yet you still havent found the issue you spoke of before.

it appears it isnt anal, its just speed x roughness to deal pain and brute

**Comment by EaglePhntm** [comment]
> > No, removing it all is lame, and yet you still havent found the issue you spoke of before.
> 
> it appears it isnt anal, its just speed x roughness to deal pain and brute

Making speed mult to 1 each should limit it to force only.

**Comment by K4rlox** [comment]
> > > No, removing it all is lame, and yet you still havent found the issue you spoke of before.
> > 
> > 
> > it appears it isnt anal, its just speed x roughness to deal pain and brute
> 
> Making speed mult to 1 each should limit it to force only.

im not a master coder, this is the only way i know to make it so it doesnt do shit
i dont want to take damage
have my chat spammed with red text
constantly cry and whimper
while using the erp menu, "just do it less rough" is not an option

File: ntf_modular/code/datums/sexcon/sexcon.dm [hunk 1]
```diff
@@ -585,24 +586,24 @@
...
+			return 0
 		if(SEX_FORCE_EXTREME)
-			return 3.0
+			return 0
 
 /datum/sex_controller/proc/get_speed_pain_multiplier(passed_speed)
 	switch(passed_speed)
 		if(SEX_SPEED_LOW)
-			return 0.8
+			return 0
```
Comment (EaglePhntm) [inline ntf_modular/code/datums/sexcon/sexcon.dm:600]: make speed mults 1 and low 0.5, it should make force the only factor,

File: ntf_modular/code/datums/sexcon/sexcon.dm [hunk 2]
```diff
@@ -585,24 +586,24 @@
 /datum/sex_controller/proc/get_force_pain_multiplier(passed_force)
 	switch(passed_force)
 		if(SEX_FORCE_LOW)
-			return 0.5
+			return 0
```
Comment (EaglePhntm) [inline ntf_modular/code/datums/sexcon/sexcon.dm:589]: revert force changes, it should be this way, whole point of force.

File: ntf_modular/code/datums/sexcon/sexcon.dm [hunk 3]
```diff
@@ -197,9 +197,9 @@
 		if(SEX_FORCE_MID)
 			oxyloss_multiplier = 0
 		if(SEX_FORCE_HIGH)
-			oxyloss_multiplier = 1.0
+			oxyloss_multiplier = 0
```
Comment (EaglePhntm) [inline ntf_modular/code/datums/sexcon/sexcon.dm:200]: No.

File: ntf_modular/code/datums/sexcon/sexcon.dm [hunk 4]
```diff
@@ -233,15 +233,16 @@
 		xuser.gain_plasma(5, TRUE)
 
 	adjust_arousal(arousal_amt)
-	damage_from_pain(pain_amt)
```
Comment (EaglePhntm) [inline ntf_modular/code/datums/sexcon/sexcon.dm:236]: revert, revert the proc below aswell.

**Comment by K4rlox** [comment]
no
you just responded to every single one of my changes and said "revert this"

i do not want to take damage, i do not want my chat spammed with this shit
image
this isnt roguetown, erp does not need to make your arms and legs bleed if you are fucking someone fastly

"just do it slowly" isnt a solution, this is

**Comment by EaglePhntm** [comment]
Bad solution, I'll do it myself the right way.

**Pull request closed without merging**